provision/testdata/list_flavors.stdout.txt

```diff
@@ -1,4 +1,5 @@
 INFO: Available configuration flavors:
+INFO: openshift-4.4-openstack:	Red Hat OpenShift Container Platform 4.4 on OpenStack
 INFO: openshift-4.4-esx:	Openshift 4.4 ESX
 INFO: openshift-4.4:	Red Hat OpenShift Container Platform 4.4
```

Comment:
I am not sure that we will use this flavor anymore:
INFO: openshift-4.4:	Red Hat OpenShift Container Platform 4.4

So perhaps hide it for now.

Reply:
Done!